mrp_production_auto_validate/models/mrp_production.py

```diff
@@ -39,12 +40,6 @@ def check_bom_auto_validate(self):
                     ).format(qty=mo.bom_id.product_qty)
                 )
 
-    @api.onchange("bom_id")
-    def _onchange_bom_id(self):
-        res = super()._onchange_bom_id()
-        self.auto_validate = self.bom_id.mo_auto_validation
-        return res
```

Comment:
It was done this way to not make the MO dependent on the BoM configuration once it is created. On the other hand I do not like to depend on `onchange` mechanism for that.

Reply:
I'll switch it to a writeable computed field